packages/twenty-front/src/modules/ui/field/display/components/LinkDisplay.tsx

```diff
@@ -2,7 +2,7 @@ import { isNonEmptyString } from '@sniptt/guards';
 import { LinkType, RoundedLink, SocialLink } from 'twenty-ui/navigation';
 
 type LinkDisplayProps = {
-  value?: { url: string; label?: string };
+  value?: { url: string | null; label?: string | null };
```

Comment:
Fine with me but here, I would force value to be defined and url to be defined too. 
We should cut the tree earlier: if value is not defined or url is not defined, there is no point rendering a LinkDisplay

Reply:
(neat)

Reply:
I made the types stricter 👍